Editor/TextProvider.cs

```diff
@@ -176,6 +176,12 @@ internal sealed class TextProvider
             { Strings.SettingsUISdkNextStepLabel, "The SDK settings are configured. You can find an usage example by going to \"(Top Menu Bar) > GameLift > Import Sample Game\", and look at \"Assets\\Scripts\\Server\\GameLiftServer.cs\"."},
             { Strings.LabelOpenSdkIntegrationDoc, "Open SDK Integration Guide"},
             { Strings.LabelOpenSdkApiDoc, "Open SDK API Reference"},
+            
+            { Strings.TabLanding, "Amazon GameLift" },
+            { Strings.TabCredentials, "AWS User Profiles" },
+            { Strings.TabAnywhere, "GameLift Anywhere" },
+            { Strings.TabEC2, "Managed EC2" },
```

Comment:
Is this correct? Mocks seems to read
* Host with Anyhwere
* Host with Managed EC2

Reply:
Strings updated